lms/djangoapps/courseware/masquerade.py

```diff
@@ -115,6 +115,11 @@ def get(self, request, course_key_string):
                     'name': 'Learner',
                     'role': 'student',
                 },
+                {
+                    'name': 'Specific Student...',
+                    'role': 'student',
+                    'user_name': request.user.username,
+                },
```

Comment:
This is the bit that toggles support on the frontend.
Items added here are used to populate the dropdown options.